apps/src/templates/codeDocs/EmbeddedBlock.jsx

```diff
@@ -11,6 +11,10 @@ export default function EmbeddedBlock({blockName, link, ariaLabel}) {
   useEffect(() => {
     if (blockName && blockRef.current) {
       const blocksDom = parseElement(`<block type='${blockName}' />`);
+      const previousBlockSpace = blockRef.current.querySelector(
```

Comment:
In the screenshot it looked like we were creating duplicate blocks. Is there a way we can just skip creating a new block rather than deleting the existing one? If not, can you add a comment here explaining why we need to do this?